templates/typescript/controller.ts

```diff
@@ -3,19 +3,12 @@
 'use strict';
 
 module <%= scriptAppName %> {
-  export interface I<%= classedName %>Scope extends ng.IScope {
-    awesomeThings: any[];
-  }
-
   export class <%= classedName %>Ctrl {
-
-    constructor (private $scope: I<%= classedName %>Scope) {
-      $scope.awesomeThings = [
+    public awesomeThings: string[] = [
```

Comment:
That's an array. [Playground link](http://www.typescriptlang.org/Playground#src=var%20x%3A%20number%5B%5D%20%3D%20%5B1%2C%202%2C%203%5D%3B%20).